Feature/keywords

aleesteele opened the pull request:
- Displays as flex, wrap
- Clicking on keyword leads to search
- added styles in `post` for 'Related Reading' and 'More on...'

jnschrag commented:
@aleesteele Something is breaking the Travis build. Can you check that out/fix and then we can do a second review?

aleesteele commented:
@jnschrag Fixed.

jnschrag commented on assets/_sass/components/_keywords.scss:
```diff
@@ -0,0 +1,35 @@
+.keywords {
+  ul {
```
Take a look at this again. The styles being assigned to the `<ul>` aren't going to be called, because this will compile to: `.keywords ul`. Since the `keywords` class is assigned to the `<ul>`, you want this instead:

```scss
.keywords {
   @extend %tag;
    margin: 0;
    padding: 0;
    padding-bottom: 0.5em;
    text-align: left;
    list-style-type: none;
}
```

aleesteele replied:
Fixed! Yes, the `ul` tag was totally ineffective... woops!

aleesteele commented:
- The text colors were a problem with the embedded styles, they should be functioning now
- Switched styling to rems, will use that from now on to be consistent. 
- Cleaned up the margins (used longhand value) for keywords for the margin property. Did the same thing for li just to be consistent.

jnschrag commented:
Fixed the build - just had to rerun it. @leezaweb This is ready for a second review.

leezaweb commented:
@jnschrag @aleesteele 
Keywords looks great. I added a bit to the bottom margin.
Question about placeholders: h2--thin is 25px in zeplin and h5--thin is 16px.... Looks like Anne created the h5 to be proportional to the h2--placeholder

jnschrag commented:
@leezaweb Good catch. The 25px on the `%h2--thin` is a mistake. It's 24px every where else it's used, so that's what is included in the placeholder.

@aleesteele It's technically invalid HTML to have an h5 after an h2 (since you're missing the hierarchy of h3, h4 coming first). Why don't you update that placeholder to be %read-more (since it's only used here) and change the element to a div instead of an h5? Just to confirm, the styles for it should be this:

```scss
  font-family: $font__lato;
  @include font-size(16px);
  font-weight: 300;
  line-height: 1.34;
  letter-spacing: 0.2px;
```

Once that's fixed, this can get approved.

aleesteele commented:
The changes have been implemented, that totally makes sense about the placeholders, especially as that style isn't being used anywhere else on the site.`h5--thin` has been changed to `read-me`. Thank you, @leezaweb and @jnschrag!
